WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -261,7 +328,7 @@ private void addUsername(EditText editText, ValidationEndListener validationEndL
 
         mUsernameButtons.put(username, usernameButton);
 
-        validateAndStyleUsername(Arrays.asList(new String[]{ username }), validationEndListener);
+        validateAndStyleUsername(Arrays.asList(new String[]{username}), validationEndListener);
```

Comment:
Since we're modifying this line, feel free to remove the String array creation, just like AS suggests (`Array.asList` is a varargs method). Thanks!